tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -38,9 +41,22 @@ export default async function (tree: Tree, schema: MigrateConvergedPkgGeneratorS
   updateLocalJestConfig(tree, options);
   updateRootJestConfig(tree, options);
 
+  // 3. setup storybook
+  setupStorybook(tree, options);
+
+  // 4. move stories to package
+  moveStorybookFromReactExamples(tree, options);
+  deleteProjectFolderInReactExamples(tree, options);
```

Comment:
Oops just realized we should move the dependency from `react-examples`

Reply:
you meant remove right ?

Reply:
yeah remove

Reply:
at least I don't think stories depend on each other currently 😨😨

Reply:
done https://github.com/microsoft/fluentui/pull/18402 thx!